src/Language/PureScript/Publish/Utils.hs

```diff
@@ -16,14 +16,20 @@ import qualified System.FilePath.Glob as Glob
 globRelative :: Glob.Pattern -> IO [FilePath]
 globRelative pat = do
   currentDir <- getCurrentDirectory
-  filesAbsolute <- Glob.globDir1 pat currentDir
+  glob pat currentDir
+
+-- | Glob relative to the specified directory, and produce relative pathnames.

```

Comment:
So the result will be a list of path names relative to the current directory? I don't understand why you might want that, why does it work like that? It's not really what I would expect given the comment it has at the moment.